dex/src/test/scala/com/wavesplatform/dex/actors/orderbook/OrderBookActorSpecification.scala

```diff
@@ -720,7 +721,7 @@ class OrderBookActorSpecification
   private def getAggregatedSnapshot(orderBookRef: ActorRef): OrderBookAggregatedSnapshot = {
     val pair = wavesUsdPair // hack
     val askAdapter = new OrderBookAskAdapter(new AtomicReference(Map(pair -> Right(orderBookRef))), 5.seconds)
-    Await.result(askAdapter.getAggregatedSnapshot(pair), 1.second).toOption.flatten.getOrElse(throw new IllegalStateException(
+    askAdapter.getAggregatedSnapshot(pair).futureValue(1.second).toOption.flatten.getOrElse(throw new IllegalStateException(
```

Comment:
can we increase this up to 5 sec?

Reply:
Why? I see that this test works fine, so don't see a problem in such a small timeout

Reply:
Suggest to remove the argument. We will use the default configuration from the base class.

Reply:
Done